cloc_multiple_directories/cloc_multiple_directories.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+
+
+def handle_arguments():
+    """Just to verify if there are two arguments"""
+    if len(sys.argv) != 3:
+        raise IndexError(
+            "Please, remember only add odoo_bin path and repo_path. Example: "
+            "python cloc_multiple_directories.py /home/odoo/odoo-15.0/odoo-bin "
+            "/home/odoo/odoo-15.0/addons/"
+        )
+    odoo_bin_path = sys.argv[1]  # /home/odoo/odoo-15.0/odoo-bin
+    repo_path = sys.argv[2]  # /home/odoo/odoo-15.0/addons/
+    return odoo_bin_path, repo_path
+
+
+def execute_odoo_cloc(odoo_bin_path, repo_path):
+    """
+    Search all subdirectories in the repo path except those which start with .
+    as .git or .github directories and apply the command odoo cloc to resume the
+    counting in a table and this method saves the result in a file called cloc_file.log
+
+    Params:
+        odoo_bin_path: str
+            full odoo-bin path
+        repo_path: str
+            full repository path to count the lines of code
+    """
+    try:
+        path_to_count = ""
+        for directory in [
+            d
+            for d in os.listdir(repo_path)
+            if os.path.isdir(os.path.join(repo_path, d)) and not d.startswith(".")
+        ]:
+            path_to_count = "%s-p %s/%s/ " % (path_to_count, repo_path, directory)
```

Comment:
instead of /, use `os.path.join`, to ensure multi-platform compatibility

Reply:
Done!